Dockerfile

```
@@ -12,12 +12,22 @@ COPY requirements.txt /app
 RUN apt-get update
 RUN apt-get install dialog apt-utils -y
 
+# Install libspatialindex   
+# See: https://github.com/libspatialindex/libspatialindex/wiki/1.-Getting-Started
+RUN apt-get install -y curl
+RUN apt-get install -y g++
+RUN apt-get install -y make
```

I think the Docker suggested way to run multiple `apt-get` commands is in one line.

- https://forums.docker.com/t/dockerfile-run-apt-get-install-all-packages-at-once-or-one-by-one/17191
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#decouple-applications

```suggestion
RUN apt-get install -y curl \
  g++ \
  make
```